Vagrantfile

```diff
@@ -87,8 +87,6 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     leader.vm.hostname = "leader"
     leader.vm.network "private_network", ip: "33.33.33.10"
 
-    leader.vm.synced_folder ".", "/vagrant", disabled: true
```

Comment:
I think it's a good idea to allow syncing this folder, it won't be used much in practice, but it's nice to have if you want to allow running `./gt-stack.py` without having to mess with dependencies.